src/routes/movies.py

```
@@ -1,13 +1,230 @@
+
+    result = await db.execute(
+        select(MovieModel)
+        .options(
+            selectinload(MovieModel.country),
+            selectinload(MovieModel.genres),
+            selectinload(MovieModel.actors),
+            selectinload(MovieModel.languages),
+        )
+        .where(MovieModel.id == movie.id)
+    )
+    movie_with_relationships = result.scalar_one()
+
+    return movie_with_relationships
+
+
+@router.get("/{movie_id}/", response_model=MovieDetailSchema)
+async def get_movie(
+    movie_id: int,
+    db: AsyncSession = Depends(get_db),
+):
+    movie = await db.scalar(
+        select(MovieModel)
+        .options(
+            joinedload(MovieModel.country),
+            joinedload(MovieModel.genres),
+            joinedload(MovieModel.actors),
+            joinedload(MovieModel.languages)
+        )
+        .where(MovieModel.id == movie_id)
+    )
+    if not movie:
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail="Movie with the given ID was not found.")
+    return movie
+
+
+@router.delete("/{movie_id}/", status_code=status.HTTP_204_NO_CONTENT)
+async def delete_movie(
+    movie_id: int,
+    db: AsyncSession = Depends(get_db),
+):
+    movie = await db.get(MovieModel, movie_id)
+    if not movie:
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail="Movie with the given ID was not found.")
+    await db.delete(movie)
+    await db.commit()
+    return None
+
+
+@router.patch("/{movie_id}/", response_model=MovieUpdateResponse)
+async def update_movie(
+    movie_id: int,
+    movie_data: MovieUpdate,
+    db: AsyncSession = Depends(get_db),
+):
+    movie = await db.get(MovieModel, movie_id)
+    if not movie:
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail="Movie with the given ID was not found.")
+
+    update_data = movie_data.model_dump(exclude_unset=True)
+    for field, value in update_data.items():
+        setattr(movie, field, value)
```

In the update_movie endpoint, only scalar fields are updated. If the PATCH request includes fields like genres, actors, languages, or country, these relationships are not updated. If your requirements specify that related entities should be updatable via PATCH, you need to add logic to handle updating these relationships. Otherwise, PATCH will silently ignore such updates, which may not be the intended behavior.